assets/css/base/_general.scss

```diff
@@ -16,7 +16,6 @@ h1,
 h2 {
   font-family: $base-title-font-family;
   font-weight: 300;
-  text-align: center;
```

Comment:
this was causing all the interior page section titles to be centered, which i think was inadvertent.

Reply:
Maybe, I think centered looks better on same pages, such as https://hex.pm/login. WDYT?

We might have to give those titles different classes...

Reply:
yes, perhaps.
I think I should not conflate this PR with the style change. I am going to pull it from this branch and we can talk about it and open a new PR just for that.